downstream/modules/platform/ref-controller-database-settings.adoc

```diff
@@ -25,6 +25,13 @@ If you are compiling Postgres against OpenSSL 3.2, your system regresses to remo
 You must restart the database server after changing the value for `shared_buffers`.
 ====
 
+Warning
+====
+If you are compiling Postgres against OpenSSL 3.2, your system regresses to remove the parameter for User during startup. You can rectify this by using the BIO_get_app_data call instead of open_get_data. Only an administrator can make these changes, but it impacts all users connected to the PostgreSQL database.
```

Comment:
for the user

Reply:
I have made the changes for both BIO_get_app_data and the second API call in the same paragraph. Thank you!

Reply:
I think this note is what Ian was referring to. I have a question. Is this note a duplication of the Warning above? Should this one be removed?